<launch>
    <arg name="velocity_topic" default="/cmd_vel"/>
    <arg name="amcl_pose_topic" default="/amcl_pose"/>
    <arg name="goal_reached_topic" default="/goal_reached"/>
    <arg name="moved_distance_topic" default="/moved_distance"/>
    <arg name="goal_distance_topic" default="/goal_distance"/>

    <arg name="linear_speed_x" default="0.1"/>
    <arg name="linear_speed_y" default="0.0"/>

    <arg name="angular_speed_x" default="0.0"/>
    <arg name="angular_speed_y" default="0.0"/>
    <arg name="angular_speed_z" default="0.0"/>

    <node pkg="mp_move_manager" type="move_straight_line" name="move_straight_line" output="screen">
        <param name="velocity_topic" value="$(arg velocity_topic)"/>
        <param name="goal_reached_topic" value="$(arg goal_reached_topic)"/>
        <param name="moved_distance_topic" value="$(arg moved_distance_topic)"/>
        <param name="goal_distance_topic" value="$(arg goal_distance_topic)"/>
        <param name="amcl_pose_topic" value="$(arg amcl_pose_topic)"/>

        <param name="linear_speed_x" value="$(arg linear_speed_x)"/>
        <param name="linear_speed_y" value="$(arg linear_speed_y)"/>
        <param name="angular_speed_x" value="$(arg angular_speed_x)"/>
        <param name="angular_speed_y" value="$(arg angular_speed_y)"/>
        <param name="angular_speed_z" value="$(arg angular_speed_z)"/>
    </node>
</launch>